### ISSUE TITLE
Unsolicited peer crash on Archive peer

### ISSUE THREAD

**Issue by mitra42** [amark/gun#611]
As requested.  stack trace

```
GUN.hijack updated msg with data = jk4bfo94TUGLgM4DgwRn obstetricsgyneco00will 13837
GUN.hijack updated msg with data = jk4bfo94TUGLgM4DgwRn gynecologyobstet00chan 15016 
Note error from fetch might be misleading especially TypeError can be Cors issue: http://dweb.me/arc/archive.org/metadata/undefined
Note error from fetch might be misleading especially TypeError can be Cors issue: http://dweb.me/arc/archive.org/metadata/undefined
/usr/local/dweb-transport/node_modules/gun/gun.js:641
                                                        if(opt.age > (now - it.was)){ return }
                                                                               ^

TypeError: Cannot read property 'was' of undefined
    at Object.<anonymous> (/usr/local/dweb-transport/node_modules/gun/gun.js:641:31)
    at Object.Type.obj.map (/usr/local/dweb-transport/node_modules/gun/gun.js:136:19)
    at Timeout._onTimeout (/usr/local/dweb-transport/node_modules/gun/gun.js:640:16)
    at ontimeout (timers.js:466:11)
    at tryOnTimeout (timers.js:304:5)
    at Timer.listOnTimeout (timers.js:267:5)
Hello wonderful person! :) Thanks for using GUN, feel free to ask for help on https://gitter.im/amark/gun and ask StackOverflow questions tagged with 'gun'!
HTTPS Server started on port 4246 with /gun 
GUN: Hikacking loading trap
_hijackFactory resolving arc/archive.org/metadata
(node:5090) Warning: N-API is an experimental feature and could change at any time.
```

**Comment by mitra42** [comment]
This was version 0.9.99991  will `npm update` and see if happens again

**Comment by amark** [comment]
Hmm, I thought this was something that might have been fixed since the version you mentioned, like I said on our call @mitra42 , but I checked the code and it looks like it could/might still fail.

I've made a new fix locally, it'll go out in the **next** release (note: the fix for `.once` went out over the weekend). Gonna close this now though or else I'll forget to when I publish.

Thanks for the stack trace, very helpful!
